StantUiIosLibrary/Classes/Cells/InternPersonCell.swift

```diff
@@ -28,9 +27,10 @@ public class InternPersonCell: UITableViewCell {
                                                             y: 0,
                                                             width: 20,
                                                             height: 20))
-    
+
     private weak var mailButton: UIButton?
     private weak var callButton: UIButton?
+    private var profileImage: RoundedImageView?
```

Comment:
Why is this not weak?

Reply:
When I put weak, I receive this message: "Instance will be immediately deallocated because property 'profileImage' is 'weak'""